tests/testthat/test-marginpadding.R

```diff
@@ -0,0 +1,37 @@
+context("Margin padding")
+
+x1 <- structure(c(1:2, NA, 4:10), .Names = c("a", "b", "c", "d", "e", "f", "g",
+"h", "i", "j"))
+x2 <- matrix(rpois(28, 5), 7, 4, dimnames=list(letters[1:7], LETTERS[1:4]))
+x2[,2] <- NA
+opt <- paste("margin.left = 0, margin.right = 0, margin.top = 0, margin.bottom = 0,",
+             "margin.autoexpand = FALSE, data.label.show = TRUE,",
+             "charting.area.fill.opacity = 0.4, charting.area.fill.color = \"red\",",
+             "x.tick.show = FALSE, y.tick.show = FALSE", collapse = "")
+charting.funcs <- c("Column", "Bar", "Pyramid", "BarMultiColor", "ColumnMultiColor",
+                    "Area", "Line")
+
+# Iterate through all combinations of charting functions and data types
+for (func in charting.funcs)
+{
+    filestem <- paste0("marginpadding-", tolower(func))
+    test_that(filestem,
+    {
+        # Create command that will create widget
+        cmd <- paste0("pp <- ", func, "(x1, ", opt, ")")
+        expect_error(suppressWarnings(eval(parse(text=cmd))), NA)
```

Comment:
Could the `suppressWarnings` wrapper be removed? It can hide some serious issues. Here it seems to hide the warning about omitting missing values, even though the plot shows the missing values with a column or bar with no width?

Reply:
I've fixed this test now to remove suppressWarnings